frontend/public/components/app.jsx

```diff
@@ -166,6 +167,10 @@ class App extends React.PureComponent {
           <LazyRoute path="/k8s/cluster/clusterrolebindings/:name/copy" exact kind="ClusterRoleBinding" loader={() => import('./RBAC' /* webpackChunkName: "rbac" */).then(m => m.CopyRoleBinding)} />
           <LazyRoute path="/k8s/cluster/clusterrolebindings/:name/edit" exact kind="ClusterRoleBinding" loader={() => import('./RBAC' /* webpackChunkName: "rbac" */).then(m => m.EditRoleBinding)} />
 
+          <Route path="/k8s/cluster/alerts" exact component={AlertsPage} />
```

Comment:
The `/k8s/` route prefix is reserved for Kubernetes resources only. Put these routes under a different prefix since they are not Kubernetes resources.

Reply:
Changed the route prefix from `/k8s/cluster/` to `/monitoring/`.